source/internal/_dispatchable.js

```diff
@@ -7,16 +7,17 @@ import _isTransformer from './_isTransformer.js';
  * object in list position (last argument). If it is an array, executes [fn].
  * Otherwise, if it has a function with one of the given method names, it will
  * execute that function (functor case). Otherwise, if it is a transformer,
- * uses transducer [xf] to return a new transformer (transducer case).
+ * uses transducer created by [transducerCreator] to return a new transformer
+ * (transducer case).
  * Otherwise, it will default to executing [fn].
  *
  * @private
  * @param {Array} methodNames properties to check for a custom implementation
- * @param {Function} xf transducer to initialize if object is transformer
+ * @param {Function} transducerCreator transducer constructor if object is transformer
```

Comment:
Very minor nit.  I would use `factory` here instead of `constructor`.  This is not a constructor function (called with `new`), so I would avoid the word to avoid confusion.

Reply:
Ok, I have changed `transducer constructor` to `transducer factory`